cmd/mayactl/app/command/snapshot/snapshot.go

```diff
@@ -32,6 +44,8 @@ func NewCmdSnapshot() *cobra.Command {
 		NewCmdSnapshotList(),
 		NewCmdSnapshotRevert(),
 	)
+	cmd.PersistentFlags().StringVarP(&options.namespace, "namespace", "n", options.namespace,
+		"namespace name, required if volume is in other then dafault namespace")
```

Comment:
_n: Fix this sentence please. Get this reviewed with @mahebbar

Reply:
done.